components/profile/ProfileDetailV3.vue

```diff
@@ -453,7 +460,7 @@ const followingConfig: ButtonConfig = {
 const unfollowConfig: ButtonConfig = {
   label: $i18n.t('profile.unfollow'),
   onClick: () => {
-    follow(false)
+    // follow(false)
```

Comment:
Is it waiting for the worker deployment?

Reply:
follow will be done in next PR